llpc/util/llpcElfWriter.cpp

```diff
@@ -631,17 +631,9 @@ void ElfWriter<Elf>::WriteToBuffer(
 
 // =====================================================================================================================
 template<class Elf>
-Result ElfWriter<Elf>::ReadFromBuffer(
-    const void* pBuffer,    // [in] Buffer to read data from
-    size_t      bufSize)    // Size of the buffer
+Result ElfWriter<Elf>::CopyFromReader(const ElfReader<Elf>& reader)
```

Comment:
Follow this style

// ====
// Comments
func(
    param1, // Comments
    param2) // Comments